src/apps/antioch_kinetic_rates.C

```diff
@@ -98,7 +101,9 @@ int main(int argc, char* argv[])
   while( T < T1 )
     { 
       Antioch::TempCache<libMesh::Real> T_cache(T);
-      
+
+      libMesh::Real rho = p0/(R_mix*T);
+
       antioch_kinetics.omega_dot( T_cache, rho, Y, omega_dot );
```

Comment:
If this is more convenient to you, let's merge it.  But from a "chemistry is tricky" standpoint, I prefer keeping all those calculations stuffed into Antioch over a hard-coded ideal gas law in GRINS.  We're not planning any ludicrously-high-pressure kinetics *now*, sure, but we might later, and it would be easy to miss an equation of state if they're not all in the same place.